css/about.css

```diff
@@ -126,101 +126,103 @@ hr {
   margin: 20px 10px 0;
 }
 
 .list-mobile {
-    width: 100%;
-    height: 100vh;
-    background: #f0f0f0;
-    display: none;
+  width: 100%;
+  height: 100vh;
+  background: #f0f0f0;
+  display: none;
 }
 
 .list-mobile.active {
-    display: flex;
+  display: flex;
 }
 
 .list-mobile ul {
-    list-style: none;
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    position: absolute;
-    left: -50px;
-    top: 45px;
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  position: absolute;
+  left: -50px;
+  top: 45px;
 }
 
 .list-mobile ul li {
-    margin: 10px 12px;
-    font-size: 30px;
-    padding: 10px 12px;
+  margin: 10px 12px;
+  font-size: 30px;
+  padding: 10px 12px;
 }
 
 .list-mobile ul li a {
-    text-decoration: none;
-    color: #272a31 ;
+  text-decoration: none;
+  color: #272a31;
 }
 
 .close {
-    font-size: 25px;
-    font-family: Verdana, Geneva, Tahoma, sans-serif;
-    position: absolute;
-    left: 60vh;
-    top: 9.9px;
-    cursor: pointer;
+  font-size: 25px;
+  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  position: absolute;
+  left: 60vh;
+  top: 9.9px;
+  cursor: pointer;
 }
 
 @media only screen and (min-width: 768px) {
-    .menu_mobile {
-      display: none;
-    }
-
-    .nav-bar {
-      justify-content: space-between;
-      align-items: center;
-      width: 100%;
-      background-color: #ffffff;
-      font-size: 22px;
-      display: flex;
-    }
-
-    .nav-bar p {
-      justify-content: start;
-      margin: 12px 10px;
-    }
-
-    .nav-list {
-      list-style: none;
-      margin: 12px 10px;
-      justify-content: space-between;
-      width: 50%;
-    }
-
-    .nav-list li a {
-      text-decoration: none;
-      color: #ec5242;
-    }
-
-    .social-nav {
-      display: flex;
-      background-color: #41434b;
-      justify-content: flex-end;
-    }
-
-    .section-social a {
-      text-decoration: none;
-      color: #f0f0f0;
-    }
-
-    .section-social {
-      margin: 20px 8px 8px;
-      width: 20%;
-      justify-content: space-between;
-    }
-
-    .imgs {
-      flex-direction: row;
-    }
+  .menu_mobile {
+    display: none;
+  }
+
+  .nav-bar {
+    justify-content: space-between;
+    align-items: center;
+    width: 100%;
+    background-color: #fff;
+    font-size: 22px;
+    display: flex;
+    height: 85px;
+  }
+
+  .nav-bar p {
+    justify-content: start;
+    margin: 12px 10px;
+  }
+
+  .nav-list {
+    list-style: none;
+    margin: 12px 10px;
+    justify-content: space-between;
+    width: 50%;
+  }
+
+  .section-social a {
+    text-decoration: none;
+    color: #f0f0f0;
+  }
+
+  .nav-list li a {
+    text-decoration: none;
+    color: #ec5242;
+  }
+
+  .social-nav {
+    display: flex;
+    background-color: #41434b;
+    justify-content: flex-end;
+  }
+
+  .section-social {
+    margin: 20px 8px 8px;
+    width: 20%;
+    justify-content: space-between;
+  }
+
+  .imgs {
```

Comment:
- [ ] As stated above,  Please follow the design provided to implement some features, Compare the two images again:

--------------------------------------Original-----------------------------------------------
![nav7](https://user-images.githubusercontent.com/55635977/128221418-d0f37680-7f60-4929-8ba6-60a84669308c.png)



--------------------------------------Yours-----------------------------------------------
![nav9](https://user-images.githubusercontent.com/55635977/128221873-01bdc2cc-beea-4e0f-b1bb-a51f2fa901c4.png)

Reply:
This issue is not well resolved 😿